### PR TITLE
Add PythonVirgoTools

### PR DESCRIPTION
**Pull request by rhyspoulton**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/PythonVirgoTools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+   name: "{{ name }}"
+   version: "{{ version }}"
+
+source:
+   url: http://software.igwn.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+   sha256: d425eab31582847ea7add85db3adf8d38253c60fbe500f6ffac0eab3d73ee136
+
+build:
+  number: 0
+  skip: true  # [win or osx]
```
Comment (dopplershift) [inline recipes/PythonVirgoTools/meta.yaml:14]: Is the reason for this the lack of `pyfd` on anything but Linux? If so, you can instead do:
```suggestion
  noarch: python
```
and we can ignore the build failures.

Comment (rhyspoulton) [reply recipes/PythonVirgoTools/meta.yaml]: Yes, both `PyCm` and `PyFd` are available on Linux. Okay done.

**Review by dopplershift — APPROVED** [review]
Looks good. @fcarbogn can you confirm you are willing to be a maintainer for this recipe?

**Comment by fcarbogn** [comment]
I confirm I  am willing to be a maintainer for this recipe

**Pull request merged**